pom.xml

```diff
@@ -17,32 +17,26 @@
 	<name>Abakus :: Root</name>
 
 	<modules>
+        <module>kontrakter</module>
 		<module>felles</module>
 		<module>prosesstask</module>
 		<module>migreringer</module>
-		<module>kodeverk</module>
-		<module>kontrakt</module>
-        <module>kontrakt-vedtak</module>
 		<module>domenetjenester</module>
 		<module>web</module>
     </modules>
 
 	<properties>
-
         <sonar.moduleKey>${project.artifactId}</sonar.moduleKey>
         <sonar.projectName>fp-abakus</sonar.projectName>
         <sonar.projectKey>navikt_fp-abakus</sonar.projectKey>
 
-        <kontrakt.java.version>25</kontrakt.java.version>
+        <abakus-kontrakt.version>2.3.4</abakus-kontrakt.version>
```

Comment:
Jeg forstår ikke helt hvordan dette fungerer.

Hvordan tar du i bruk nyeste versjon i abakus? Altså project-versjon. 
Si du gjør en endring i kontrakter og ønsker å ta de i bruk, hvordan går man frem da?


Gjøres release da fra branch med manuell jobb elns?

Reply:
Vi trenger ikke noe branch - man pusher kontrakt endringer til master - appen bruker fortsatt 2.3.4 versjon av kontrakten. Så releaser man ny kontrakt (f.eks 2.4.0) og så jobber man videre med appen (eller kanskje man har endringene allerede liggende på en branch). Dvs at man bruker to steg ved kontraktendringer.